wordle-web/src/main.ts

```diff
@@ -20,6 +19,43 @@ const vuetify = createVuetify({
     sets: {
       mdi
     }
+  },
+  theme: {
+    defaultTheme: 'dark',
+    themes: {
+      christmas: {
+        colors: {
+          correct: '#346633',
+          misplaced: '#D3D3D3',
+          wrong: '#c54245',
+          primary: '#c54245'
+        }
+      },
+      halloween: {
+        colors: {
+          correct: '#E66C2C',
+          misplaced: '#D3D3D3',
+          wrong: '#121212',
+          primary: '#E66C2C'
+        }
+      },
+      dark: {
+        colors: {
+          correct: '#4CAF50',
+          misplaced: '#FFC107',
+          wrong: '#F44336',
+          primary: '#000000'
+        }
+      },
+      inverse: {
```

Comment:
I think this would be considered an additional theme and wouldn't count as the light theme since the correct and wrong colors are swapped - hence the name "inverse". So you may want to either change the colors back and call this light theme, or create an actual light theme.